community/keyutils/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+for patch in *.patch; do
+    patch -p1 < "$patch"
+done
+
+make \
+    CPPFLAGS="$CPPFLAGS -I." \
```

Comment:
Why `-I.`?

Reply:
respects the default value in the Makefile. Build directory needs to be searched for headers.

Reply:
The build system itself doesn't handle this?

Reply:
No, only accepts environment CFLAGS and LDFLAGS.